libs/shared/mocking/src/graphql/createResolvers.ts

```diff
@@ -13,6 +13,7 @@ export type ResolverMap = Record<
   | Record<string, ((...args: any) => any) | undefined>
   | GraphQLScalarType
   | undefined
+  | any
```

Comment:
I don't like that we're adding `any` types across the codebase. I understand why, but maybe we can take a minute and find better fixes for some of them.